Gruntfile.js

```diff
@@ -505,6 +510,7 @@ module.exports = function(grunt) {
     'minify',
     'karma:unit_ci',
     'nodeunit',
+    'mochaTest:unit',
```

Comment:
This should be in the `unit` target too, so we pick up on regressions locally.

Reply:
Good spot!